README.md

```diff
@@ -169,13 +169,57 @@ Stop your local postgres:
 sh docker/run-postgres.sh stop
 ```
 
+### To run Sam locally
+
+#### Local setup
+
+Set up configs using the [firecloud-develop quick start guide for configs](https://github.com/broadinstitute/firecloud-develop#quick-start---how-do-i-set-up-my-configs).
+
+If you haven't already, add `127.0.0.1       local.broadinstitute.org` to `/etc/hosts`:
+```
+sudo sh -c "echo '127.0.0.1       local.broadinstitute.org' >> /etc/hosts"
+```
+
+You can then start Sam against dev DBs or local DBs following the instructions below.
+
+#### Using dev DBs
+You must be connected to the Broad Internal network to connect to the Dev DBs.
+
+```
+# Start up local Sam
+sh config/docker-rsync-local-sam.sh
+```
+
+#### Using local DBs
+
+```
+# Start up local opendj and postgres
+sh docker/run-opendj.sh start
+sh docker/run-postgres.sh start 
```

Comment:
the above 2 steps are not required, in local mode the script starts up its own opendj and postgres

Reply:
ah ok. I see that in the script now